tianshou/policy/base.py

```diff
@@ -66,6 +67,11 @@ def __init__(
         super().__init__()
         self.observation_space = observation_space
         self.action_space = action_space
+        self.action_type = ""
+        if isinstance(action_space, (Discrete, MultiDiscrete, MultiBinary)):
+            self.action_type = "discrete"
+        elif isinstance(action_space, Box):
+            self.action_type = "continuous"
```

Comment:
I think `gym.spaces` is more than 4 types. Would `self.action_type == ""` cause any problems?

Reply:
"" eventually handles Dict/Tuple or other types.